src/go/rpk/pkg/adminapi/api_debug.go

```diff
@@ -18,11 +18,12 @@ import (
 )
 
 const (
-	debugEndpoint          = "/v1/debug"
-	selfTestEndpoint       = debugEndpoint + "/self_test"
-	cpuProfilerEndpoint    = debugEndpoint + "/cpu_profile"
-	DiskcheckTagIdentifier = "disk"
-	NetcheckTagIdentifier  = "network"
+	debugEndpoint           = "/v1/debug"
+	selfTestEndpoint        = debugEndpoint + "/self_test"
+	cpuProfilerEndpoint     = debugEndpoint + "/cpu_profile"
+	DiskcheckTagIdentifier  = "disk"
+	NetcheckTagIdentifier   = "network"
+	CloudcheckTagIdentifier = "cloud"
```

Comment:
the go/rpk changes should go into the second commit related to rpk changes.

Reply:
Agreed, fixed